ui/app/AppLayouts/Wallet/services/dapps/DAppsRequestHandler.qml

```diff
@@ -484,13 +484,11 @@ SQUtils.QObject {
 
             // Beware, the tx values are standard blockchain hex big number values; the fees values are nim's float64 values, hence the complex conversions
             if (!!tx.maxFeePerGas && !!tx.maxPriorityFeePerGas) {
-                let maxFeePerGasDec = root.store.hexToDec(tx.maxFeePerGas)
-                const gasPriceInWei = BigOps.fromString(maxFeePerGasDec)
+                maxFeePerGas = hexToGwei(tx.maxFeePerGas)
+                maxPriorityFeePerGas = hexToGwei(tx.maxPriorityFeePerGas)
+
+                // TODO: check why we need to set gasPrice here and why if it's not checked we cannot send the tx and fees are unknown????
                 gasPrice = hexToGwei(tx.maxFeePerGas)
```

Comment:
Not sure why `gasPrice` is used in this context (refer to TODO in the line above)?